public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_log_analytics/azure_log_analytics_datasource.ts

```diff
@@ -218,7 +218,7 @@ export default class AzureLogAnalyticsDatasource extends DataSourceWithBackend<
    * And some of the azure internal data sources return null in this function, which the
    * external interface does not support
    */
-  metricFindQueryInternal(query: string): Promise<MetricFindValue[]> {
+  metricFindQueryInternal(query: string, optionalOptions?: any): Promise<MetricFindValue[]> {
```

Comment:
I know elsewhere types this as `any`, but do you think we could use `unknown` instead?

It's just a spicier `any` that forces us to be more careful when interacting with it. Would love to only reduce the amount of `any`s in the code 😌